src/fdcache.cpp

```diff
@@ -1930,12 +1930,12 @@ fsblkcnt_t FdManager::GetFreeDiskSpace(const char* path)
     S3FS_PRN_ERR("could not get vfs stat by errno(%d)", errno);
     return 0;
   }
-  return (vfsbuf.f_bavail * vfsbuf.f_bsize);
+  return (vfsbuf.f_bavail * vfsbuf.f_frsize);
```

Comment:
We should multiply `f_bavail` by `f_bsize` or `f_favail` by `f_frsize` but not mix block and fragment units.  Instead do you want to cast one of these to `uint64_t`?